Explorer/Assets/Scripts/ECS/StreamableLoading/DeferredLoading/Tests/ConcurrentLoadingBudgetProviderShould.cs

```diff
@@ -0,0 +1,61 @@
+using ECS.Prioritization.DeferredLoading;
+using NUnit.Framework;
+using System.Reflection;
+
+namespace ECS.StreamableLoading.DeferredLoading.Tests
+{
+    [TestFixture]
+    public class ConcurrentLoadingBudgetProviderShould
+    {
+        [Test]
+        public void BudgetSpent()
```

Comment:
Please change the names to the infinitive form: after should always an infinitive follows